src/io/base.js

```diff
@@ -11,6 +11,10 @@ export class IOBase {
     this.path = packageOrDirPath;
     this.files = {};
     this.entries = [];
+    // If this is too large the node process will hit a RangeError
+    // when it runs out of memory.
+    let maxMegabytes = 100;
```

Comment:
Seems like this should be a default we set in constants.

Might also make sense as a CLI flag but that can be a later thing :smile: